src/mdast2jcr/partials/emphasis.js

```diff
@@ -14,10 +14,21 @@ import Handlebars from 'handlebars';
 
 function emphasis(context) {
   const uniqueName = Handlebars.helpers.nameHelper.call(context, 'button');
-
   const { children: [child] } = context;
-  const { children: [{ value }] } = child;
-  return `<button${uniqueName} sling:resourceType="core/franklin/components/button/v1/button" jcr:primaryType="nt:unstructured" link="${child.url}" linkTitle="${child.title !== null ? child.title : ''}" linkText="${value}" linkType="secondary" />`;
+  const { title, url, children: [{ value }] } = child;
+  const attributes = {
+    'sling:resourceType': 'core/franklin/components/button/v1/button',
+    'jcr:primaryType': 'nt:unstructured',
+    link: url,
+    linkText: value,
+    linkType: 'secondary',
+  };
+  if (title !== null) {
```

Comment:
I think we want to use `!=` which should cover both null and undefined, otherwise we're only checking to see if the value is actually null and destructing the property that doesn't exist should have an undefined value.

Reply:
fixed